### DISCUSSION TITLE
想請問如何在MobaXterm中連接本機資料夾

### DISCUSSION THREAD

**Discussion by Chieh-Yu2002** [bol-edu/HLS-SOC-Discussions#24] [Q&A] [answered]
想請問Lab1的workbook中，在上傳.bit/.hwh至PYNQ的這項步驟，如何將MobaXterm與本機連結?
<img width="538" alt="螢幕擷取畫面 2022-10-01 222159" src="https://user-images.githubusercontent.com/99870909/193414084-d3afe0fc-ef81-485f-9828-7a893fb71243.png">
有嘗試額外開一個SFTP，但點任何資料夾都毫無反應。
<img width="729" alt="螢幕擷取畫面 2022-10-01 222822" src="https://user-images.githubusercontent.com/99870909/193414301-9517ffa4-526e-4f8f-88ad-ce6c19e6459f.png">

**Comment by ciwade870307** [comment, marked as answer]
PYNQ目前不支援ssh的傳輸，請透過web打開jupyter後再上傳.bit/.hwh
![image](https://user-images.githubusercontent.com/50269115/193435072-f5e20b06-35a0-4413-8745-bbb16d8df70d.png)

**Reply by Chieh-Yu2002** [reply]
了解，謝謝!
